dipy/segment/tests/test_metric.py

```diff
@@ -174,7 +174,7 @@ def dist(self, v1, v2):
             if metric.are_compatible(f1.shape, f2.shape):
                 distance = metric.dist(f1, f2)
                 if np.all(f1 == f2):
-                    assert_equal(distance, 0.)
+                    assert_almost_equal(distance, 0.)
```

Comment:
Is it obvious why the distance would not be exactly 0 if the vectors are exactly equal on win32?